src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.test.ts

```diff
@@ -1,46 +1,34 @@
 import { describe, expect, it } from 'vitest'
 
+import { UTXO_MAXIMUM_BYTES_LENGTH } from '../constants'
 import { addAggregatorAndDestinationToMemo } from './addAggregatorAndDestinationToMemo'
 import { MEMO_PART_DELIMITER } from './constants'
 
 const RECEIVE_ADDRESS = '0x32DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb6'
 const AGGREGATOR_ADDRESS = '0xd31f7e39afECEc4855fecc51b693F9A0Cec49fd2'
-const FINAL_ASSET_ADDRESS = '0x8a65ac0E23F31979db06Ec62Af62b132a6dF4741'
 const AGGREGATOR_TWO_LAST_CHARS = 'd2'
+const REALLY_BIG_ADDRESS =
+  '0x32DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb632DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb6'
 
-describe('addAggregatorAndDestinationToMemo', () => {
-  it('should add aggregator address, destination address and minAmountOut correctly', () => {
-    const minAmountOut = '9508759019'
-    const affiliateBps = '100'
-    const expectedL1AmountOut = '42' // we don't care about this for the purpose of tests
-    const quotedMemo = `=:ETH.ETH:${RECEIVE_ADDRESS}:${expectedL1AmountOut}:ss:${affiliateBps}:${AGGREGATOR_TWO_LAST_CHARS}:${FINAL_ASSET_ADDRESS}:10`
+const FINAL_ASSET_ADDRESS = '0x8a65ac0E23F31979db06Ec62Af62b132a6dF4741'
+const SHORTENED_FINAL_ASSET_ADDRESS = 'a6df4741'
 
-    const slippageBps = 100 // 1%
+const COLLIDING_FINAL_ASSET_ADDRESS = '0x8a65ac0E23F31979db06Ec62Af62b132i6dF4741'
 
-    const modifiedMemo = addAggregatorAndDestinationToMemo({
-      sellChainId: ethChainId,
-      quotedMemo,
-      aggregator: AGGREGATOR_ADDRESS,
-      finalAssetAddress: FINAL_ASSET_ADDRESS,
-      minAmountOut,
-      slippageBps,
-      finalAssetPrecision: 9,
-      chainNamespace: CHAIN_NAMESPACE.Evm,
-    })
+const FINAL_ASSETID = `eip155:1/erc20:${FINAL_ASSET_ADDRESS}`
+const COLLIDING_FINAL_ASSETID = `eip155:1/erc20:${COLLIDING_FINAL_ASSET_ADDRESS}`
 
-    expect(modifiedMemo).toBe(
-      `=${MEMO_PART_DELIMITER}e${MEMO_PART_DELIMITER}${RECEIVE_ADDRESS}${MEMO_PART_DELIMITER}${expectedL1AmountOut}${MEMO_PART_DELIMITER}ss${MEMO_PART_DELIMITER}${affiliateBps}${MEMO_PART_DELIMITER}${AGGREGATOR_TWO_LAST_CHARS}${MEMO_PART_DELIMITER}${FINAL_ASSET_ADDRESS}${MEMO_PART_DELIMITER}941367103`,
-    )
-  })
+const DEFAULT_THORCHAIN_ASSETS_LIST = [FINAL_ASSETID, COLLIDING_FINAL_ASSETID]
 
-  it('should add aggregator address, destination address and minAmountOut correctly with a bigger precision', () => {
-    const minAmountOut = '2083854765519275828179229'
-    const affiliateBps = '100'
-    const expectedL1AmountOut = '42' // we don't care about this for the purpose of tests
-    const quotedMemo = `=:ETH.ETH:${RECEIVE_ADDRESS}:${expectedL1AmountOut}:ss:${affiliateBps}:${AGGREGATOR_TWO_LAST_CHARS}:${FINAL_ASSET_ADDRESS}:10`
+const affiliateBps = '100'
+const expectedL1AmountOut = '42' // we don't care about this for the purpose of tests
```

Comment:
Do we want to? i.e L1 MinOut could be large enough and impactful on the memo length

Reply:
Though that's probably the responsibility of `makeMemoWithShortenedFinalAssetAmount` tests so probably fine

Reply:
Agree, this one isn't shortening against the maximum data length, it would be the responsibility of `makeMemoWithShortenedFinalAssetAmount`, let's keep it as it is